dbux-code/src/globalAnalysisView/nodes/GlobalErrorsChildNode.js

```diff
@@ -8,4 +8,11 @@ export default class GlobalErrorsChildNode extends TraceNode {
   buildChildren() {
     return this.childTraces.map(t => this.treeNodeProvider.buildNode(TraceNode, t, this));
   }
+
+  getSelectedChildren() {
```

Comment:
Should this function not be in `TraceNode`?

Reply:
`TraceNode`s have their own `isSelected` method. `getSelectedChildren` is for their parent but not `TraceNode` themself.